lib/robots/dor_repo/release/release_members.rb

```diff
@@ -17,16 +17,19 @@ def perform_work
           logger.debug "release-members working on #{druid}"
 
           return unless cocina_object.collection?
+          return unless add_wf_to_members?
 
-          publish_collection
+          members.each { |member| create_release_workflow(member) }
         end
 
         private
 
-        def publish_collection
-          member_service = Dor::Release::MemberService.new(druid:)
-          add_workflow_to_members(member_service.items) if add_wf_to_members?
-          add_workflow_to_members(member_service.sub_collections)
+        def members
+          object_client.members.select { |member| published?(member) }
```

Comment:
The published check was previously performed by DSA. However, it is not performed here as it may take some time (since getting from workflow service instead of solr).